packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -355,5 +355,21 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       name: "beacon_weak_head_detected_total",
       help: "Detected current head block is weak. May reorg it out when proposing next slot. See proposer boost reorg for more",
     }),
+
+    // Validator activity metrics
+    validatorActivity: {
```

Comment:
might wanna put them to the other state/chain metrics above, seems like at the bottom we have the non-spec'ed metrics